avocado/plugins/runner_nrunner.py

```diff
@@ -67,6 +67,16 @@ def initialize(self):
                                  metavar="HOST:PORT",
                                  help_msg=help_msg)
 
+        help_msg = ('Buffer size that status server uses.  This should '
+                    'generally not be a concern to most most users, but '
+                    'it can be tunned in case a runner generates very large '
+                    'status messages. Defaults is 33554432 (32MiB)')
+        settings.register_option(section=section,
+                                 key='status_server_buffer_size',
```

Comment:
I hope one day we will concentrate all status server configs on 'nrunner.status_server' (or even 'status_server') namespace. But since all other configs are on 'nrunner' ns, I'm ok with that.

Reply:
I agree, and I was bothered by the lengthy key too.